src/commands/flutter-symbols/upload.ts

```diff
@@ -119,7 +119,7 @@ export class UploadCommand extends Command {
 
     this.config = await resolveConfigFromFile(this.config, {
       configPath: this.configPath,
-      defaultConfigPath: DEFAULT_CONFIG_PATH,
+      defaultConfigPaths: ['datadog-ci.json'],
```

Comment:
Why not use `DEFAULT_CONFIG_PATHS` here?

Reply:
I don't want to create any regression on other commands - if you're fine with `'datadog-ci.json'`, having `'../datadog-ci.json'` will only create regressions.
If you think that `DEFAULT_CONFIG_PATHS` should rather be `['datadog-ci.json']`, that also makes sense to me and I can change it

Reply:
I think it'd makes sense to keep `DEFAULT_CONFIG_PATHS = ['datadog-ci.json']`, use it everywhere and have `react-native` be the exception with `['datadog-ci.json', '../datadog-ci.json']`